keyboards/satan/keymaps/olligranlund_iso/rules.mk

```
@@ -13,5 +13,5 @@ MIDI_ENABLE = no            # MIDI controls
 AUDIO_ENABLE = no           # Audio output on port C6
 UNICODE_ENABLE = no         # Unicode
 BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
-RGBLIGHT_ENABLE = yes        # Enable WS2812 RGB underlight. 
+RGBLIGHT_ENABLE = yes        # Enable WS2812 RGB underlight.  Do not enable this with audio at the same time.
```

This warning is incorrect, and has not been valid for a long time. 

```suggestion
RGBLIGHT_ENABLE = yes        # Enable WS2812 RGB underlight. 
```

Eg, you can absolutely enable both functions at the same time.